delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/encoder/JsonLdEncoder.scala

```diff
@@ -63,6 +63,19 @@ trait JsonLdEncoder[A] {
       ntriples <- IO.fromEither(graph.toNTriples)
     } yield ntriples
 
+  /**
```

Comment:
Since you added this, can you then change the `def dot` and `def ntriples` to use this method?

```scala
  def dot(
      value: A
  )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, Dot] =
    for {
      graph    <- graph(value)
      dot      <- graph.toDot(context(value))
    } yield dot

  def ntriples(
      value: A
  )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, NTriples] =
    for {
      graph    <- graph(value)
      ntriples <- IO.fromEither(graph.toNTriples)
    } yield ntriples
```